tools/flang2/flang2exe/ll_structure.c

```diff
@@ -591,11 +591,14 @@ ll_create_module(const char *module_name, const char *target_triple,
 
 void
 add_linker_directives(LLVMModuleRef module) {
+#if LLVM_VERSION_MAJOR < 5
```

Comment:
Instead of using the LLVM version like this, maybe we should send the version from `flang` so that it is checked at runtime.

Reply:
I would personally prefer not to support LLVM < 5.0. Initializing the debug info here is questionable and not sure what the full implications are. But not sure sending the command line flags is a good idea, because we're building against LLVM anyway (I'd prefer to keep the binary "clean" in the newer version).

Reply:
> because we're building against LLVM anyway

No, we are not. Flang is independent of the LLVM version. I just found out that `flang.exe` sends the LLVM version to `flang2.exe` and some parts of code use that. I've update the PR.